<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>data_toolbox</class>
 <widget class="QWidget" name="data_toolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Toolbox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_block_size">
     <property name="text">
      <string>Block Size</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_file">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QRadioButton" name="radio_button_volume">
     <property name="toolTip">
      <string>The whole data is stored in a single dataset, similar to MSA0309.</string>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="radio_button_slice_by_slice">
     <property name="toolTip">
      <string>Each slice is represented as a separate dataset, similar to Vervet1818.</string>
     </property>
     <property name="text">
      <string>Slice by slice</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_file_type">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="4">
    <widget class="QLineEdit" name="line_edit_direction">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The path to the direction dataset. For &quot;slice by slice&quot; type data, %Slice% acts as a reserved keyword indicating the slice number dataset.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>%Slice%/Microscope/Processed/Registered/Direction</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_datasets">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Datasets</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="QLineEdit" name="line_edit_block_size">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The attribute which specifies the number of grouped voxels in each axis (distribution only).</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>BlockSize</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="4">
    <widget class="QLineEdit" name="line_edit_retardation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The path to the retardation dataset. For &quot;slice by slice&quot; type data, %Slice% acts as a reserved keyword indicating the slice number dataset.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>%Slice%/Microscope/Processed/Registered/Retardation</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="button_browse">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="5">
    <spacer name="spacer_vertical">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>168</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="QLineEdit" name="line_edit_transmittance">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The path to the transmittance dataset. For &quot;slice by slice&quot; type data, %Slice% acts as a reserved keyword indicating the slice number dataset.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>%Slice%/Microscope/Processed/Registered/Transmittance</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_vector_spacing">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>Vector Spacing</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_retardation">
     <property name="text">
      <string>Retardation</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_direction">
     <property name="text">
      <string>Direction</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_attributes">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Attributes</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_transmittance">
     <property name="text">
      <string>Transmittance</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="line_edit_file">
     <property name="toolTip">
      <string>The system path to the root file.</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>C:/Test.h5</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="spacer_horizontal">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>31</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="QLineEdit" name="line_edit_vector_spacing">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The attribute which specifies the distance between two voxels in each axis.</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>VectorSpacing</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_file_path">
     <property name="text">
      <string>Path</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="4">
    <widget class="QLineEdit" name="line_edit_inclination">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The path to the inclination dataset. For &quot;slice by slice&quot; type data, %Slice% acts as a reserved keyword indicating the slice number dataset.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>%Slice%/Microscope/Processed/Registered/Inclination</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_distribution">
     <property name="text">
      <string>Distribution</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="4">
    <widget class="QLineEdit" name="line_edit_distribution">
     <property name="toolTip">
      <string>The path to the distribution (spherical harmonics) dataset. For &quot;slice by slice&quot; type data, %Slice% acts as a reserved keyword indicating the slice number dataset.</string>
     </property>
     <property name="placeholderText">
      <string>%Slice%/Microscope/Processed/Registered/Distribution</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_inclination">
     <property name="text">
      <string>Inclination</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radio_button_slice_by_slice</tabstop>
  <tabstop>radio_button_volume</tabstop>
  <tabstop>line_edit_file</tabstop>
  <tabstop>button_browse</tabstop>
  <tabstop>line_edit_vector_spacing</tabstop>
  <tabstop>line_edit_block_size</tabstop>
  <tabstop>line_edit_transmittance</tabstop>
  <tabstop>line_edit_retardation</tabstop>
  <tabstop>line_edit_direction</tabstop>
  <tabstop>line_edit_inclination</tabstop>
  <tabstop>line_edit_distribution</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
